### ISSUE TITLE
Failed prop type: PaymentCardTextField: prop type `style` is invalid; it must be a function, usually from the `prop-types` package, but received `object`

### ISSUE THREAD

**Issue by ajaykumar97** [tipsi/tipsi-stripe#440]
## The problem
I am getting the following warning in the debugger console:

`Warning: Failed prop type: PaymentCardTextField: prop type `style` is invalid; it must be a function, usually from the `prop-types` package, but received `object`.`

## Environment
* `tipsi-stripe` version: 7.3.0
* Last `tipsi-stripe` version where the issue was not reproduced (if applicable): 5.6.0
* iOS or Android:  iOS
* OS version: 11.4
* React-Native version: 0.57.8

## Code To Reproduce Issue (Good To Have)

```
<View
  style={{
    borderBottomWidth: 1,
    borderColor: 'orange',
    marginBottom: 5
 }}
>
     <PaymentCardTextField
        style={{
          borderBottomWidth: 1,
          borderColor: 'red'
       }} 
       numberPlaceholder={'4242 4242 4242 4242'}
       expirationPlaceholder={'MM/YYYY'}
       cvcPlaceholder={'CVC'}
      disabled={false}
      onParamsChange={this.handleFieldParamsChange}
     />
</View>
```

**Comment by vTrip** [comment]
Also receiving this warning

**Comment by PetteriHaro** [comment]
Same here. Any solution to this?

**Comment by ajaykumar97** [comment]
I found an alternative for ``PaymentCardTextField`` : [react-native-credit-card-input](https://github.com/sbycrosz/react-native-credit-card-input)

**Comment by quachsimon** [comment]
I am also having this issue. It's odd because I have this component in two separate areas of my app. One part that renders this component does not produce the warning whereas the other part does. What could be the issue causing this?

**Comment by ajaykumar97** [comment]
@quachsimon The reason for receiving warning only once is because of the reason that (as per my observation) in react-native, similar type of warning are printed only once i.e. multiple same warning get merged into once.
As I have mentioned in the above comment, you can make use of the [react-native-credit-card-input](https://github.com/sbycrosz/react-native-credit-card-input) for horizontal card number input and [react-native-credit-card-input-fullpage](https://www.npmjs.com/package/react-native-credit-card-input-fullpage) for vertical card number input.

**Comment by yassinecc** [comment]
Should be fixed by https://github.com/tipsi/tipsi-stripe/pull/477

**Comment by ajaykumar97** [comment]
Sorry for the late reply. Thanks @yassinecc. Your solution fixed the problem.

**Comment by bramski** [comment]
Please make a release. There have been no releases since april.
